### PR TITLE
CMP-2336 Make CO 1.4.1 Release Notes Live

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s): 4.12+

Issue:  https://errata.engineering.redhat.com/advisory/129828

Link to docs preview: https://75011--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html

QE review:
- [X] QE has approved this change.

Additional information: This PR merely uncomments the content generated previously for the CO 1.4.1 Release Notes.  QE approval was given for that PR. https://github.com/openshift/openshift-docs/pull/74715

### CODE REVIEW

**Review by sheriff-rh — APPROVED** [review]
Changes look good - I'll merge when the errata is live.

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* Previously, when a profile was removed from a completed scan shown in `ScanSettingBinding` (SSB), Compliance Operator did not remove the old scan. Afterward, when launching a new SSB using the deleted profile, Compliance Operator failed to update the result. With this release of Compliance Operator, the new SSB now shows the new compliance check result. (link:https://issues.redhat.com/browse/OCPBUGS-29272[*OCPBUGS-29272*])
+
+* Previously, on `ppc64le`, the metrics service was not created. With this update, when deploying the Compliance Operator v1.4.1 on ppc64le, the metrics service is now created correctly. (link:https://issues.redhat.com/browse/OCPBUGS-32797[*OCPBUGS-32797*])
+
+* Previously, on a HyperShift hosted cluster, a scan with ocp4-pci-dss profile will run into fatal error due to `filter cannot iterate`. With this release, the scan for `ocp4-pci-dss will reach `done` status and return either a `Compliance` or `Non-Compliance` test result. (link:https://issues.redhat.com/browse/OCPBUGS-33067[*OCPBUGS-33067*])
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc]: I have corrected this - good job, vale!

**Comment by GroceryBoyJr** [comment]
Comment to peer reviewer: I've added two new release note entries to the 1.4.1 release, which is due to be delivered May 8th.

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 2]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+== OpenShift Compliance Operator 1.4.1
+
+The following advisory is available for the OpenShift Compliance Operator 1.4.1:
+
+* link:https://access.redhat.com/errata/RHBA-2024:1830[RHBA-2024:1830 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* As of this release, Compliance Operator now provides the CIS OpenShift 1.5.0 profile rules.  (link:https://issues.redhat.com/browse/CMP-2447[*CMP-2447*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:28]:
```suggestion
* As of this release, the Compliance Operator now provides the CIS OpenShift 1.5.0 profile rules. (link:https://issues.redhat.com/browse/CMP-2447[*CMP-2447*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 3]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* With this update, Compliance Operator now provides `OCP4 STIG ID` and `SRG` with the profile rules. (link:https://issues.redhat.com/browse/CMP-2401[*CMP-2401*])
+
+* With this update, obsolete rules being applied to `s390x` have been removed. (link:https://issues.redhat.com/browse/CMP-2471[*CMP-2471*])
+
+[id="compliance-operator-1-4-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, for RHCOS systems using RHEL9, application of the `ocp4-kubelet-enable-protect-kernel-sysctl-file-exist` rule failed. This update replaces the rule with `ocp4-kubelet-enable-protect-kernel-sysctl`. Now, after auto remediation is applied, RHEL9-based RHCOS systems will show `PASS` upon the application of this rule. (link:https://issues.redhat.com/browse/OCPBUGS-13589[*OCPBUGS-13589*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:37]: Suggest using common attributes:
```
:op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)
:op-system: RHCOS
:op-system-lowercase: rhcos
:op-system-base: RHEL
:op-system-base-full: Red Hat Enterprise Linux (RHEL)
```
```suggestion
* Previously, for {op-system-first} systems using {op-system-base-full} 9, application of the `ocp4-kubelet-enable-protect-kernel-sysctl-file-exist` rule failed. This update replaces the rule with `ocp4-kubelet-enable-protect-kernel-sysctl`. Now, after auto remediation is applied, {op-system-base} 9-based {op-system} systems will show `PASS` upon the application of this rule. (link:https://issues.redhat.com/browse/OCPBUGS-13589[*OCPBUGS-13589*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 4]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* Previously, after applying compliance remediations using profile `rhcos4-e8`, the nodes were no longer accessible using SSH to the core user account. With this update, nodes remain accessible through SSH using sshkey. (link:https://issues.redhat.com/browse/OCPBUGS-18331[*OCPBUGS-18331*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:39]: Wondering if this should be:
```suggestion
* Previously, after applying compliance remediations using profile `rhcos4-e8`, the nodes were no longer accessible using SSH to the core user account. With this update, nodes remain accessible through SSH using an SSH key. (link:https://issues.redhat.com/browse/OCPBUGS-18331[*OCPBUGS-18331*])
```
or
```suggestion
* Previously, after applying compliance remediations using profile `rhcos4-e8`, the nodes were no longer accessible using SSH to the core user account. With this update, nodes remain accessible through SSH using the `sshkey` option. (link:https://issues.redhat.com/browse/OCPBUGS-18331[*OCPBUGS-18331*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 5]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* Previously, the `STIG` profile was missing rules from CaC that fulfill requirements on the published `STIG` for {product-title}. With this update, upon remediation, the cluster satisfies `STIG` requirements that can be remediated using Compliance Operator. (link:https://issues.redhat.com/browse/OCPBUGS-26193[*OCPBUGS-26193*])
+
+* Previously, creating a `ScanSettingBinding` with profiles of different types for multiple products bypassed a restriction against multiple products types in a binding. With this update, the product validation now allows multiple products regardless of the of profile types in the `ScanSettingBinding`. (link:https://issues.redhat.com/browse/OCPBUGS-26229[*OCPBUGS-26229*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:43]:
```suggestion
* Previously, creating a `ScanSettingBinding` object with profiles of different types for multiple products bypassed a restriction against multiple products types in a binding. With this update, the product validation now allows multiple products regardless of the of profile types in the `ScanSettingBinding` object. (link:https://issues.redhat.com/browse/OCPBUGS-26229[*OCPBUGS-26229*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 6]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+The following advisory is available for the OpenShift Compliance Operator 1.4.1:
+
+* link:https://access.redhat.com/errata/RHBA-2024:1830[RHBA-2024:1830 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* As of this release, Compliance Operator now provides the CIS OpenShift 1.5.0 profile rules.  (link:https://issues.redhat.com/browse/CMP-2447[*CMP-2447*])
+
+* With this update, Compliance Operator now provides `OCP4 STIG ID` and `SRG` with the profile rules. (link:https://issues.redhat.com/browse/CMP-2401[*CMP-2401*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:30]:
```suggestion
* With this update, the Compliance Operator now provides `OCP4 STIG ID` and `SRG` with the profile rules. (link:https://issues.redhat.com/browse/CMP-2401[*CMP-2401*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 7]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* With this update, instructions for the use of the `rhcos4-banner-etc-issue` rule are enhanced to provide more detail. (link:https://issues.redhat.com/browse/OCPBUGS-28797[*OCPBUGS-28797*])
+
+* Previously the `api_server_api_priority_flowschema_catch_all` rule provided `FAIL` status on {product-title} 4.16 clusters. With this update, the `api_server_api_priority_flowschema_catch_all` rule provides `PASS` status on {product-title} 4.16 clusters. (link:https://issues.redhat.com/browse/OCPBUGS-28918[*OCPBUGS-28918*])
+
+* Previously, when a profile was removed from a completed scan shown in `ScanSettingBinding` (SSB), Compliance Operator did not remove the old scan. Afterward, when launching a new SSB using the deleted profile, Compliance Operator failed to update the result. With this release of Compliance Operator, the new SSB now shows the new compliance check result. (link:https://issues.redhat.com/browse/OCPBUGS-29272[*OCPBUGS-29272*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:51]:
```suggestion
* Previously, when a profile was removed from a completed scan shown in a `ScanSettingBinding` (SSB) object, the Compliance Operator did not remove the old scan. Afterward, when launching a new SSB using the deleted profile, the Compliance Operator failed to update the result. With this release of the Compliance Operator, the new SSB now shows the new compliance check result. (link:https://issues.redhat.com/browse/OCPBUGS-29272[*OCPBUGS-29272*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 8]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* Previously, on `ppc64le`, the metrics service was not created. With this update, when deploying the Compliance Operator v1.4.1 on ppc64le, the metrics service is now created correctly. (link:https://issues.redhat.com/browse/OCPBUGS-32797[*OCPBUGS-32797*])
+
+* Previously, on a HyperShift hosted cluster, a scan with ocp4-pci-dss profile will run into an unrecoverable error due to `filter cannot iterate`. With this release, the scan for `ocp4-pci-dss will reach `done` status and return either a `Compliance` or `Non-Compliance` test result. (link:https://issues.redhat.com/browse/OCPBUGS-33067[*OCPBUGS-33067*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:55]:
```suggestion
* Previously, on a HyperShift hosted cluster, a scan with the `ocp4-pci-dss` profile will run into an unrecoverable error due to a "filter cannot iterate" issue. With this release, the scan for the `ocp4-pci-dss` profile will reach `done` status and return either a `Compliance` or `Non-Compliance` test result. (link:https://issues.redhat.com/browse/OCPBUGS-33067[*OCPBUGS-33067*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 9]
```diff
@@ -15,39 +15,45 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* Previously the `api_server_api_priority_flowschema_catch_all` rule provided `FAIL` status on {product-title} 4.16 clusters. With this update, the `api_server_api_priority_flowschema_catch_all` rule provides `PASS` status on {product-title} 4.16 clusters. (link:https://issues.redhat.com/browse/OCPBUGS-28918[*OCPBUGS-28918*])
+
+* Previously, when a profile was removed from a completed scan shown in `ScanSettingBinding` (SSB), Compliance Operator did not remove the old scan. Afterward, when launching a new SSB using the deleted profile, Compliance Operator failed to update the result. With this release of Compliance Operator, the new SSB now shows the new compliance check result. (link:https://issues.redhat.com/browse/OCPBUGS-29272[*OCPBUGS-29272*])
+
+* Previously, on `ppc64le`, the metrics service was not created. With this update, when deploying the Compliance Operator v1.4.1 on ppc64le, the metrics service is now created correctly. (link:https://issues.redhat.com/browse/OCPBUGS-32797[*OCPBUGS-32797*])
```
Comment (adellape) [inline security/compliance_operator/compliance-operator-release-notes.adoc:53]:
```suggestion
* Previously, on `ppc64le` architecture, the metrics service was not created. With this update, when deploying the Compliance Operator v1.4.1 on `ppc64le` architecture, the metrics service is now created correctly. (link:https://issues.redhat.com/browse/OCPBUGS-32797[*OCPBUGS-32797*])
```

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 2 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:28]: Done, thank you!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 4 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:39]: I chose the second option - thank you @adellape

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 6 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:30]: Thank you!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 7 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:51]: thank you!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 8 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:55]: oh lordy thank you.

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 9 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:53]: Yes, thank you.

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 5 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:43]: Thank You!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 3 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:37]: Done, thank you.

**Pull request merged**